fetch.bs

```diff
@@ -4668,6 +4706,9 @@ steps. They return a <a for=/>response</a>.
 
     <p>is true; otherwise false.
 
+   <li><p>If <a>Cross-Origin-Embedder-Policy allows credentials</a> with <var>request</var> returns
+   false, set <var>includeCredentials</var> to false.</p>
```

Comment:
```suggestion
   false, then set <var>includeCredentials</var> to false.</p>
```

Reply:
Done in https://github.com/whatwg/fetch/pull/1229/commits/89aaca1fdd377c5b10133cceb56a100ca618fd27